File: src/test/intellibitz/sted/ui/FontMapperDesktopFrameTest.java
package intellibitz.sted.ui;

import org.junit.Before;
import org.junit.Test;

public class FontMapperDesktopFrameTest {
    private STEDWindow stedWindow;

    public FontMapperDesktopFrameTest() {
    }

    @Before
    public void testSTEDWindow() {
//        stedWindow = new STEDWindow();
//        stedWindow.load();
    }

    @Test
    public void testFontMapperDesktopFrame() {
        DesktopFrame desktopFrame =
                new DesktopFrame();
        desktopFrame.init();
    }

}

File: src/test/intellibitz/sted/ui/FontMapperDesktopFrameTest.kt
package sted.ui

import kotlin.test.Test

class FontMapperDesktopFrameTest {
    private val stedWindow: STEDWindow? = null
//    @Before
    fun testSTEDWindow() {
//        stedWindow = new STEDWindow();
//        stedWindow.load();
    }

    @Test
    fun testFontMapperDesktopFrame() {
        val desktopFrame = DesktopFrame()
        desktopFrame.init()
    }
}